apps/new-dealer/views/weapons-ammunition.html

```diff
@@ -1,14 +1,15 @@
 {{<partials-page}}
   {{$page-content}}
-
-    <div class="panel-indent">
-      <p>{{#t}}pages.{{weaponsOrAmmunition}}.important{{/t}}</p>
-    </div>
-
-    {{#fields}}
-      {{#renderField}}{{/renderField}}
-    {{/fields}}
-
-    {{#input-submit}}continue{{/input-submit}}
+  <div>
+    <p>{{#t}}pages.{{weaponsOrAmmunition}}.helper{{/t}}</p>
+  </div>
+  <div class="panel-indent">
```

Comment:
Can’t remember has panel-indent been replaced in govuk?

Reply:
I have replaced this with  “govuk-inset-text” per govuk design system.